src/main/java/atmosfears/AtmosFearsBE/controller/DataIngestController.java

```diff
@@ -0,0 +1,39 @@
+package atmosfears.AtmosFearsBE.controller;
+
+import atmosfears.AtmosFearsBE.model.AirParticulatesFile;
+import atmosfears.AtmosFearsBE.service.AirParticulatesService;
+import atmosfears.AtmosFearsBE.service.CalculationService;
+import atmosfears.AtmosFearsBE.service.DataLoadService;
+import java.util.Set;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@RequestMapping("/api/ingest")
```

Comment:
```suggestion
@RequestMapping("/api/ingestion-data")
```
Don't use verbs in REST endpoints. If this PR is related to the #3 issue and the responsibility of this endpoint is to save data from sensors, you can name this endpoint, e.g. `sensors-measurements` or simply `measurements`.